docs/pages/ktexttruncator.vue

```diff
@@ -15,6 +15,35 @@
         </p>
       </DocsShow>
     </DocsPageSection>
+    <DocsPageSection
+      title="Usage"
+      anchor="#usage"
+    >
+      <p>
+        Use <code>KTextTruncator</code> to shorten long text while keeping it readable. It limits
```

Comment:
This is already mentioned in the "Overview" section. Generally we try to avoid repeating same information unless it's really needed. I'd suggest we remove this paragraph from here and just keep the one in the "Overview", however you're welcome to improve the existing "Overview" text.